pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -269,3 +281,11 @@ func (c *TargetConfigController) namespaceEventHandler() cache.ResourceEventHand
 		},
 	}
 }
+
+func backVarsToArgs(vars map[string]string) string {
+	var args []string
+	for k, v := range vars {
+		args = append(args, fmt.Sprintf("- --%s=%s", k, v))
+	}
+	return strings.Join(args, "\n")
+}
```

Comment:
I'm not entirely sure yet whether we could/should add this into the `BackupConfig` struct itself, maybe that's a bit better if it's centralized

Reply:
👍🏽

Reply:
moved. should `BackupConfig` be `Singleton` ?

actually if it is, then using a global mutex wont be an issue, it is actually better iiuc

Reply:
moved 👍🏽

Reply:
done 👍🏽